docs-ref-services/latest/activedirectory.md

```diff
@@ -16,7 +16,11 @@ ms.devlang: nodejs
 ## Overview
 
 > [!IMPORTANT]
-> Use [Microsoft Graph](https://graph.microsoft.io/) instead of Azure AD Graph API to access Azure Active Directory resources. Our development efforts are now concentrated on Microsoft Graph and no further enhancements are planned for Azure AD Graph API. For more information, see [Migrate Azure AD Graph apps to Microsoft Graph](https://docs.microsoft.com/graph/migrate-azure-ad-graph-overview).
+> To help developers take advantage of all the identity features available in our platform, we recommend that you use the [Microsoft Authentication Library (MSAL)](https://docs.microsoft.com/azure/active-directory/develop/reference-v2-libraries) and the [Microsoft Graph API](https://docs.microsoft.com/graph/overview) in your application development. We are no longer adding any new features to ADAL and Azure AD Graph. We will continue to provide technical support and security updates but will no longer provide feature updates.
```

Comment:
Recommend cutting this way back to be shorter/more direct. Get in "ADAL is being deprecated" and "use MSAL instead" and you're good.

Here are the dates:

https://docs.microsoft.com/azure/active-directory/develop/msal-migration#frequently-asked-questions-faq

> **Q: Is ADAL being deprecated?**
> A: Yes. Starting June 30th, 2020, we will no longer add new features to ADAL. We'll continue adding critical security fixes to ADAL until June 30th, 2022. After this date, your apps using ADAL will continue to work, but we recommend upgrading to MSAL to take advantage of the latest features and to stay secure.

Reply:
Addressed